### PR TITLE
[SDPA-5452] CMS review page pagination

### PR DESCRIPTION
**Pull request by mayurngondhkar**
** Jira ** - https://digital-engagement.atlassian.net/browse/SDPA-5452

Changes:

Change the Diff modules revision_pager_limit to 16 in settings

image

### CODE REVIEW

File: tide_core.install [hunk 1]
```diff
@@ -1371,3 +1371,17 @@ function tide_core_update_8057() {
...
 }
+
+/**
+ * Changes the diff modules general_settings.revision_pager_limit to 16.
+ */
+function tide_core_update_8058() {
+  if (\Drupal::moduleHandler()->moduleExists('diff')) {
+    $config = \Drupal::configFactory()
+      ->getEditable('diff.settings');
+    $value = $config->get('general_settings.revision_pager_limit');
```
Comment (anthony-malkoun) [inline tide_core.install:1382]: If this value is not set, does this mean `$value` will be falsey? Or are we only setting it if it has some other value?

**Review by anthony-malkoun — COMMENTED** [review]
I think you might want to check that `$config` exists rather than if `$value` does.

**Comment by mayurngondhkar** [comment]
> I think you might want to check that `$config` exists rather than if `$value` does.

Added the check.

Stumbled across this resource  https://www.computerminds.co.uk/drupal-code/loading-config-programmatically

File: tide_core.install (hunk 1 above)
Comment (mayurngondhkar) [reply tide_core.install]: Idea was to check if $value was not empty.

**Review by anthony-malkoun — APPROVED** [review]
Looking good. Approved. Thanks for explaining the default being 50 and the link to https://www.computerminds.co.uk/drupal-code/loading-config-programmatically

**Pull request merged**